src/state/opamUpdate.ml

```diff
@@ -162,6 +162,55 @@ let repository rt repo =
             }
         ))
 
+let get_sys_available rt =
+  let merged_depexts_opams =
+    let depexts_equal (pkgset1, filter1) (pkgset2, filter2) =
+      OpamSysPkg.Set.equal pkgset1 pkgset2 &&
+      OpamTypesBase.filter_equal filter1 filter2
+    in
+    OpamRepositoryName.Map.fold (fun _ opams acc ->
+        OpamPackage.Map.union (fun x y ->
+            let depexts_x = OpamFile.OPAM.depexts x in
+            let depexts_y = OpamFile.OPAM.depexts y in
+            if OpamCompat.List.equal depexts_equal depexts_x depexts_y then
+              x
+            else
+              OpamFile.OPAM.with_depexts (depexts_x @ depexts_y) x)
+          acc opams)
+      rt.repo_opams OpamPackage.Map.empty
+  in
+  let repo_depexts =
+    OpamFileTools.get_depexts merged_depexts_opams
+      ~env:(OpamPackageVar.resolve_global rt.repos_global)
+  in
+  if not (OpamFile.Config.depext rt.repos_global.config) then
```

Comment:
Does these  `if not (OpamFile.Config.depext rt.repos_global.config) then` handle setting no depexts via cli or environnement variable ?